README.md

```diff
@@ -37,6 +37,8 @@ plugins: [
  - `optionsObj.functionName`: the default value is `__`, you can change it to other function name.
  - `optionsObj.failOnMissing`: the default value is `false`, which will show a warning message, if the mapping text cannot be found. If set to `true`, the message will be an error message.
  - `optionsObj.hideMessage`: the default value is `false`, which will show the warning/error message. If set to `true`, the message will be hidden.
+ - `optionsObj.pluralIdentName`: the default value is `count`, which will be used to determine whether key is plural or not.
```

Comment:
Possible to merge this a one option e.g `options.plurals`

```js
options.plurals = {
   identName: 'count',
   ruleNumber:  1
}
```

Are `identName` && `ruleNumber` standardize names for plurals ?

Reply:
I'm not certain about names. Just used something that is related to the stuff :)